docs_nnx/why.rst

```diff
@@ -327,14 +339,14 @@ implement a LoRA layer and replace a Linear layer of an existing model with it.
 
   ...
 
-As should above, in Linen this doesn't really work in this case because the ``.linear`` submodule
-is not available, however the rest of the code gives an idea how the ``params`` structure must be
-manually updated.
+As shown above, in Flax Linen this doesn't really work in this case because the ``linear`` sub-``Module``
+is not available. However, the rest of the code provides an idea of how the ``params`` structure must be manually updated.
 
-Performing arbitrary model surgery is not very easy in Flax Linen, currently the
+Performing arbitrary model surgery is not easy in Flax Linen, and currently the
 `intercept_methods <https://flax-linen.readthedocs.io/en/latest/api_reference/flax.linen/module.html#flax.linen.intercept_methods>`_
-API is the only was to do generic patching of methods but it's not very ergonomic. In NNX, using ``iter_graph`` its very easy
-to do generic model surgery, below is an example of replacing all Linear layers in a model with LoRA layers.
+API is the only way to do generic patching of methods. But this API is not very ergonomic.
+
+In Flax NNX, to do generic model surgery you can just use ``nnx.iter_graph``, which is much simpler and easier than in Linen. Below is an example of replacing all ``nnx.Linear`` layers in a model with custom-made ``LoraLinear`` NNX layers.
```

Comment:
maybe instead:

"...much easier than the Linen `intercept_methods` described above..."

@IvyZX @cgarciae